Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/WindowsMixedRealityUtilities.cs

```diff
@@ -51,6 +50,8 @@ private static SpatialCoordinateSystem GetSpatialCoordinateSystem(IntPtr nativeP
         }
 #endif //ENABLE_DOTNET
 
+        public static IWindowsMixedRealityUtilitiesProvider WmrUtilitiesProvider { get; set; } = null;
```

Comment:
Would there be any other utilities provider other than Windows Mixed Reality? If not, maybe name this as UtilitiesProvider for ease of reading?

Reply:
For this static class as it's currently named and written, no. This class is specific to Windows Mixed Reality. I can still rename it though, since it's a little redundant.

Reply:
Renamed to `UtilitiesProvider`